test/integration/api.bats

```diff
@@ -111,9 +111,24 @@ function teardown() {
 	skip
 }
 
-# FIXME
 @test "docker logs" {
-	skip
+	start_docker 3
+	swarm_manage
+
+	# run a container with echo command
+	run docker_swarm run -d --name test_container busybox echo hello world
+	[ "$status" -eq 0 ]
+
+	# make sure container exists
+	run docker_swarm ps -l
+	[ "${#lines[@]}" -eq 2 ]
+	[[ "${lines[1]}" ==  *"test_container"* ]]
+
+	# verify
+	run docker_swarm logs test_container
+	[ "$status" -eq 0 ]
+	[ "${#lines[@]}" -eq 1 ]
+	[[ "${lines[0]}" ==  *"hello world"* ]]
```

Comment:
We could probably add a second part here to test `logs` with multiple lines output.